.github/workflows/build_docker.yaml

```diff
@@ -21,13 +21,13 @@ jobs:
               src_image: "rocker/r-ver:4.0.4", 
               cran_url: "https://packagemanager.rstudio.com/cran/2021-04-20",
               tag: "rbmi:r404",
-              mmrm_version : "v0.13"
+              mmrm_version : "v0.1.3"
```

Comment:
can already increase to v0.1.5 now - just had some minor fixes

Reply:
ty will update (this will be removed once mmrm hits the public rspm servers)